pkg/apis/scorecard/v1alpha3/configuration_types.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 The Operator-SDK Authors
```

Comment:
Can we remove the duplicative `TestSpec` type?

https://github.com/estroz/operator-sdk/blob/3d7fdb526a56af231cfd9c458df8eacd6bbe8d71/pkg/apis/scorecard/v1alpha3/test_types.go#L33-L41

And consolidate to use `TestConfiguration`?